src/helper.sh

```diff
@@ -79,16 +87,16 @@ write_to_log() {
 	dest_dir=$1
 	dest_file=$2
 	dest_path=$dest_dir$dest_file
-	
+
 	shift # eat first argument
 	shift # eat second argument
-	
+
 	# create empty file (overwrite previous content if it exists)
-	> $dest_path
-	
+	: > "$dest_path"
+
 	# append remaining args to file
-	for i in "$*"; do
-		cat <<< "$i" >> $dest_path
+	for i in "$@*"; do
```

Comment:
I was curious what the @ does. Would you care to explain?
(You can ignore this comment)